api/serializers/company_serializers.py

```diff
@@ -6,7 +6,7 @@
 class CompanySerializer(serializers.ModelSerializer):
     class Meta:
         model = Company
-        fields = ['id', 'created', 'name', 'location', 'slogan', 'logo_url']
+        fields = ['id', 'created', 'name', 'location', 'slogan', 'logo_url', 'email']
```

Comment:
There will be a merge conflict here when https://github.com/countryrudd/api.devlink.tech/pull/20  is merged, just giving a warning.